pkg/tfgen/docs.go

```diff
@@ -455,6 +455,21 @@ const (
 	sectionImports             = 5
 )
 
+func (p *tfMarkdownParser) parseSupplementaryExamples() (string, error) {
+	examplesFileName := fmt.Sprintf("docs/%s/%s.md", p.kind, p.rawname)
```

Comment:
As discussed, consider making the files `*.examples.md` to be more future-proof.